xapian-core/backends/glass/glass_dbcheck.cc

```diff
@@ -738,6 +738,16 @@ check_glass_table(const char * tablename, const string &db_dir, int fd,
 		continue;
 	    }
 
+	    // Check doclen with doclen lower and upper bounds
+	    if (doclen < version_file.get_doclength_lower_bound() ||
+		doclen > version_file.get_doclength_upper_bound()) {
+		if (out) {
+		    *out << "doclen not within bounds";
```

Comment:
Thanks.

This looks good, except that this really should write `endl` too so that the message is on its own line rather than running into whatever is output next.  Existing similar messages do this.